src/OpenTelemetry.Instrumentation.Http/README.md

```diff
@@ -72,30 +72,32 @@ For an ASP.NET application, adding instrumentation is typically done in the
 ## Advanced configuration
 
 This instrumentation can be configured to change the default behavior by using
-`HttpClientInstrumentationOptions` (.NET/.NET Core applications) or
-`HttpWebRequestInstrumentationOptions` (.NET Framework applications). It is
-important to note that even if `HttpClient` is used in .NET Framework
-applications, it underneath uses `HttpWebRequest`. Because of this,
-`HttpWebRequestInstrumentationOptions` is the configuration option for .NET
-Framework applications, irrespective of whether `HttpWebRequest` or `HttpClient`
-is used.
+`HttpClientInstrumentationOptions`. It is important to note that there are
+differences between .NET Framework and newer .NET/.NET Core runtimes which
+govern what options are used. On .NET Framework, `HttpClient` uses the
+`HttpWebRequest` API. On .NET & .NET Core, `HttpWebRequest` uses the
```

Comment:
```suggestion
`HttpWebRequest` API. On .NET & .NET Core, `HttpClient` uses the
```

Reply:
yea i can see this is not easy to articulate. Lets merge this now, and come back to fixing the wording.